seo/README.md

```diff
@@ -0,0 +1,14 @@
+# SEO Coding Culture
+
+## Overview
+
+This is a collection of documents that outline the SEO teams culture and developer expectations.
```

Comment:
Maybe change `SEO teams culture and developer expectations` to `SEO development culture & expectations`. It is not just the SEO team that should be using these guidelines. Also the culture repo is open to the public who might not be aware of our team make up all the docs should be easy to read for those outside the holiday extras group.